pkg/cluster/k8sres.go

```diff
@@ -969,6 +975,10 @@ func (c *Cluster) getPodEnvironmentConfigMapVariables() ([]v1.EnvVar, error) {
 func (c *Cluster) getPodEnvironmentSecretVariables() ([]v1.EnvVar, error) {
 	secretPodEnvVarsList := make([]v1.EnvVar, 0)
 
+	if len(c.Spec.Env) > 0 {
+		secretPodEnvVarsList = append(secretPodEnvVarsList, c.Spec.Env...)
+	}
+
 	if c.OpConfig.PodEnvironmentSecret == "" {
 		return secretPodEnvVarsList, nil
 	}
```

Comment:
What is the expected outcome of  this piece of code when one has in the manifest
```yaml
spec:
  env:
...
  - name: minio_secret_key
      valueFrom:
        secretKeyRef:
          name: my-custom-secret
          key: minio_secret_key
```
but ` c.OpConfig.PodEnvironmentSecret` is not set ?

Reply:
This should return `[]v1.EnvVar` from the Postgresql CR. i.e. 
```
  - name: minio_secret_key
      valueFrom:
        secretKeyRef:
          name: my-custom-secret
          key: minio_secret_key
```